ur5_moveit_config/package.xml

```diff
@@ -5,7 +5,7 @@
   <description>
      An automatically generated package with all the configuration and launch files for using the ur5 with the MoveIt Motion Planning Framework
   </description>
-  <author email="fxm@ipa.fhg.de">Felix Messmer</author>
+  <author>Felix Messmer</author>
   <maintainer email="fxm@ipa.fhg.de">Felix Messmer</maintainer>
```

Comment:
any special reason to keep the maintainer  only for `ur5_moveit_config` ?

Reply:
No, seems I missed that one.

Should be fixed now in fce99c1.